src/styles.css

```diff
@@ -1089,3 +1089,39 @@
     max-width: 138px;
   }
 }
+
+.results-matched-options-container {
+  --grid-template-columns: repeat(3, minmax(80px, 33%));
```

Comment:
This creates a weird space in certain cases. What do you think about making this `flex` and left aligning it?
![image](https://github.com/Constructor-io/constructorio-ui-quizzes/assets/2954126/90bc507f-94a0-4a45-86bb-7c459dc20b86)